generate_test.go

```diff
@@ -204,3 +204,29 @@ $GENERATE 32-158 dhcp-${-32,4,d} A 10.0.0.$
 		}
 	}
 }
+
+
+
+func TestCrasherString(t *testing.T) {
+	tests := []struct{
+	in  string
+	err string
+}{
+		{"$GENERATE 0-300103\"$$GENERATE 2-2", "dns: Expect blank after $GENERATE range: \"\\\"\" at line: 1:19"},
+		{"$GENERATE 0-5414137360", "dns: Expect blank after $GENERATE range: \"\\n\" at line: 1:22"},
+		{"$GENERATE 0-5414137360", "dns: Expect blank after $GENERATE range: \"\\n\" at line: 1:22"},
+		{"$GENERATE       11522-3668518066406258", "dns: Expect blank after $GENERATE range: \"\\n\" at line: 1:38"},
```

Comment:
not that this type of high range number would crash NewRR with
```
runtime: goroutine stack exceeds 1000000000-byte limit
fatal error: stack overflow
```

if the rest of the input was legit.